test/instrumentation/levee/main.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/google/go-flow-levee/pkg/levee"
+	"golang.org/x/tools/go/analysis/singlechecker"
+)
+
+func main() {
+	singlechecker.Main(levee.Analyzer)
```

Comment:
does go-flow-levee not provide a binary for this upstream?

Reply:
It does, but I was struggling to get it to play nicely with the `go.mod`.  But with your other comments, it looks like I missed a useful pattern in that regard.  I'll investigate shortly.